modules/nf-core/bowtie2/align/main.nf

```diff
@@ -38,6 +38,7 @@ process BOWTIE2_ALIGN {
     }
 
     def samtools_command = sort_bam ? 'sort' : 'view'
+    def extension = args2.contains("--output-fmt") ? (args2 =~ /--output-fmt+\s+(\S+)/)[0][1].toLowerCase() : ""
```

Comment:
`--output-fmt` also has a short form `-O` can you also make sure this accounts for that too?